go.mod

```diff
@@ -96,13 +96,16 @@ require (
 	github.com/shirou/gopsutil/v3 v3.22.8
 	github.com/stretchr/testify v1.8.2
 	go.etcd.io/bbolt v1.3.6
+	go.opentelemetry.io/otel v1.15.0-rc.1
```

Comment:
We have to use the rc because there are import errors that are fixed in the RC. Link? Can we add a todo/jira to follow up once we have a stable release.

Reply:
Actually it got released https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.15.1
🥳 So I'll update this!

Reply:
Updated!